inference_rules.adoc

```diff
@@ -129,6 +129,21 @@ MLCommons shall retain a library of past audit reports and send copies to MLComm
 
 An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline can be requested to be invalidated by the auditee. The final decision to accept such a request will be taken by the Working Group.
 
+=== Late Submissions and Late Upload of Logs
+
+To ensure fairness to all submitters and the integrity of the process, MLPerf Inference will strictly adhere to the https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#late-submissions[MLCommons general submission rules]. The 60 minute grace period will be strictly enforced, and any late submissions will not be considered as valid submissions. Exceptions will be made only in extraordinary circumstances as listed in the https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#post-submission-extension-for-extraordinary-circumstances-submission-deadline-72-hours[mlcommons guidelines].
+
+Once the submission deadline has passed, the submission round enters the review phase. During this period, submitters are **not permitted to upload new accuracy, performance, or power logs**. 
+
+**Exceptions** to this rule may be granted solely at the discretion of the Review Committee, and only to ensure compliance with the established rules. For instance, if logs are corrupted during the submission process or if the submission checker fails to detect missing logs, the Review Committee may authorize a re-upload of the affected logs.
+
+**No modifications** to hyperparameters, batch size, loadgen configurations and flags, dataset sorting, or any other performance-sensitive parameters are allowed after the submission deadline. This restriction applies even if such changes are necessary to comply with the rules.
+
+Under no circumstances will submitters be allowed to upload new logs that alter the accuracy, performance, or power results, regardless of the extent of the change.
```

Comment:
Can we add a 1-2% delta here to account for noise - if the change is asked by the review committee?